instance/templates/instance/ansible/swift.yml

```diff
@@ -0,0 +1,6 @@
+EDXAPP_SWIFT_AUTH_VERSION: '2'
+EDXAPP_SWIFT_USERNAME: '{{ user }}'
+EDXAPP_SWIFT_KEY: '{{ password }}'
+EDXAPP_SWIFT_TENANT_NAME: '{{ tenant }}'
+EDXAPP_SWIFT_AUTH_URL: '{{ auth_url }}'
+EDXAPP_SWIFT_REGION_NAME: '{{ region }}'
```

Comment:
They are both belonging to @omarkhan:

    PR#11871: Custom icons for SSO ... (omarkhan) - open-craft/omar/sso-icon-images (2fc7721)
    PR#11854: Allow for tooltips with ... (omarkhan) - open-craft/omar/rich-tooltips (d673de3)

The first PR is merged, the second isn't.  Anyway, even if we got rid of those, we don't want to break the persistent databases feature, do we?